base/applications/taskmgr/taskmgr.c

```diff
@@ -482,14 +482,19 @@ TaskManagerWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
         TrayIcon_UpdateIcon();
         break;
 
-    case WM_ENTERMENULOOP:
-        TaskManager_OnEnterMenuLoop(hDlg);
+    case WM_INITMENUPOPUP:
+        /* Do not disable the status bar if we opened the system menu */
+        if (!HIWORD(lParam))
+            TaskManager_DisableStatusBar(hDlg);
+        else
+            TaskManager_EnableStatusBar(hDlg);
```

Comment:
I think you need to preserve `WM_ENTERMENULOOP` handler and move `bInMenuLoop` variable assignment there.

```suggestion
            TaskManager_EnableStatusBar(hDlg);
        break;
    case WM_ENTERMENULOOP:
        bInMenuLoop = TRUE;
        break;
```